src/pages/index.js

```diff
@@ -2,76 +2,215 @@ import "./index.css";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import Section from "../components/Section.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/UserInfo.js";
-import { initialCards, config } from "../utils/constants.js";
+import { config } from "../utils/constants.js";
+import Api from "../components/API.js";
 
 const profileEditButton = document.querySelector("#profile-edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const cardAddButton = document.querySelector(".profile__add-button");
 const addModal = document.querySelector("#profile-add-card");
-
 const profileEditForm = profileEditModal.querySelector("#edit-modal-form");
 const nameInput = document.querySelector("#profile-title-input");
 const jobInput = document.querySelector("#profile-description-input");
 const cardAddForm = addModal.querySelector("#add-modal-form");
-const cardListEl = document.querySelector(".cards__list");
+const profilePicture = document.querySelector("#profile-change-button");
+const pictureChangeModal = document.querySelector("#profile-change-picture");
+const pictureChangeForm = pictureChangeModal.querySelector(
+  "#picture-modal-form"
+);
+const deleteFormButton = document.querySelector(".cards__delete-button");
 
 const cardTemplateSelector = "#card-template";
 
-const editCardModal = new PopupWithForm(
-  "#profile-edit-modal",
-  handleProfileEditSubmit
-);
-editCardModal.setEventListeners();
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authToken: "6a75e942-1cdf-4570-be93-f1316f67106a",
+});
 
-const previewImageModal = new PopupWithImage("#preview-image-modal");
-previewImageModal.setEventListeners();
-const userInfo = new UserInfo(".profile__title", ".profile__description");
-const itemSection = new Section({
-  items: initialCards,
-  renderer: renderCard,
-  cssSelector: ".cards__list",
+let itemSection;
+
+api.getCardList().then((items) => {
+  itemSection = new Section({
+    items,
+    renderer: renderCard,
+    cssSelector: ".cards__list",
+  });
+
+  itemSection.renderItems(items);
 });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**

Reply:
You haven't corrected my remark https://github.com/MrWatson0531/se_project_aroundtheus/pull/1#discussion_r1577849182